pkg/guest/util.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2023 cuisongliu@qq.com.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package guest
+
+import (
+	"fmt"
+
+	"github.com/labring/sealos/pkg/constants"
+)
+
+func FormalizeWorkingCommand(clusterName string, imageName string, t string, cmd string) string {
+	if cmd == "" {
+		return ""
+	}
+	switch t {
+	case constants.RootfsImage, constants.PatchImage:
```

Comment:
<img width="525" alt="image" src="https://github.com/labring/sealos/assets/4150905/0906cf4f-893c-4159-86d6-468a6f20614e">
using this define ?

Reply:
done